experiments/process_sandbox/tests/sandboxlib-upcall-basic.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+
+#include "cxxapi/cxxsandbox.h"
+#include "sandbox.hh"
+#include "shared.h"
+
+int test(int idx)
+{
+  int v = 12;
```

Comment:
Ah, here's why 12!! Ok, the fact that this is in another file confuses people. Perhaps a comment?